src/extras.py

```diff
@@ -58,7 +58,12 @@ def augment_with_salary(record: Record) -> str:
 def _get_oo_id_mapping() -> Mapping[str, str]:
     if OO_CACHE.currsize == 0:
         log.info("Refreshing OO ID cache")
-        response = requests.get(OO_ID_SOURCE_URL)
+        try:
+            response = requests.get(OO_ID_SOURCE_URL, timeout=60)
+            response.raise_for_status()
+        except Exception as err:
+            log.warning(f"OO ID URL {OO_ID_SOURCE_URL} failed with error: {err}")
```

Comment:
`log.exception` might be more useful here

Reply:
Great point!

Reply:
Actually, I'd like to leave this as a `warning`. It's not critical for displaying the record, it just means that the OO ID wasn't able to be found for the record. Warning conveys that a bit better.

Reply:
Should we add something like "No OO ID found, if you have more information about this officer please contact us" or something to that effect.

Reply:
Sure, I can do that

Reply:
This is for when we're retrieving information on _all_ officers, so I don't think that'd be an appropriate step to add here. Currently that section just won't get rendered if we don't have an entry for a specific officer.